python/ray/scripts/scripts.py

```diff
@@ -149,20 +149,36 @@ def cli():
     is_flag=True,
     default=None,
     help="use the raylet code path")
+@click.option(
+    "--quiet",
+    is_flag=True,
+    default=False,
+    help="do not print debugging startup messages")
+@click.option(
+    "--no-redirect-worker-output",
+    is_flag=True,
+    default=False,
+    help="do not redirect worker stdout and stderr to files")
+@click.option(
+    "--no-redirect-output",
+    is_flag=True,
+    default=False,
+    help="do not redirect non-worker stdout and stderr to files")
 def start(node_ip_address, redis_address, redis_port, num_redis_shards,
           redis_max_clients, redis_shard_ports, object_manager_port,
           object_store_memory, num_workers, num_cpus, num_gpus, resources,
           head, no_ui, block, plasma_directory, huge_pages, autoscaling_config,
-          use_raylet):
+          use_raylet, quiet, no_redirect_worker_output, no_redirect_output):
     # Convert hostnames to numerical IP address.
     if node_ip_address is not None:
         node_ip_address = services.address_to_ip(node_ip_address)
     if redis_address is not None:
         redis_address = services.address_to_ip(redis_address)
 
     if use_raylet is None and os.environ.get("RAY_USE_XRAY") == "1":
-        # This environment variable is used in our testing setup.
-        print("Detected environment variable 'RAY_USE_XRAY'.")
+        if not quiet:
+            # This environment variable is used in our testing setup.
+            print("Detected environment variable 'RAY_USE_XRAY'.")
```

Comment:
I think we probably want to do this by using the `logging` module and then set the logging level to something like `warn` if `--quiet` is passed in (or possibly even let people pass in `--logging-level=warn).

See https://github.com/ray-project/ray/blob/079c4e482acd1300d65505543d1511b141e9bd30/python/ray/worker.py#L28

and 

https://github.com/ray-project/ray/blob/079c4e482acd1300d65505543d1511b141e9bd30/python/ray/services.py#L31-L35

Reply:
cc @richardliaw

Reply:
Sure, but this could be done in some future PR. :-)

Also, BTW, I am not sure if the fact that you are blindly reconfiguring `logging` on import of the `ray.services` Python module is against some policy, but it should be. Now I cannot configure `logging` myself my way.

I think you should configure logging inside `init` and not at the top-level of the module.

I have found a workaround which is `ray_services.logger.removeHandler(ray_services.ch)`. :-)

But yea, I completely agree you should use `logging`. Only I would prefer if this PR is merged now and you later on move to `logging` as you see fit.

Reply:
Also, you should probably not use the same `logger` for all files. A good practice is to do `logger = logging.getLogger(__name__)` at the top of every file. Then one can filter things based on the source file.

Reply:
I think @robertnishihara is right. It is not necessary to add the `quiet` argument. We need to setup the logging level setting.